File: oct_27/student/Student.java
public class Student extends Grade {
  String name;
  int rollNo;
  
  public Student(String name, int rollNo, int marks) {
    this.name = name;
    this.rollNo = rollNo;
    this.marks = marks;
  }

  public String display() {
    return String.format(
      "Student Name: %s%nStudent Roll No: %d%nStudent Marks: %d (Grade: %s)%n",
      name,
      rollNo,
      marks,
      calculateGrade()
    );
  }

  @Override
  public String toString() {
    return String.format(
      "Student(name=%s, rollNo=%i, marks=%i)",
      name,
      rollNo,
      marks
    );
  }
}

File: nov_17/student.py
class Student:
  def __init__(self, roll_no, marks):
    self.roll_no = roll_no
    self.marks = marks

  def is_passed(self):
    return self.marks > 40

stud = Student(25, 50)

print("Student Roll No:", stud.roll_no)
print("Student Marks:", stud.marks)
print("Student Passed?", stud.is_passed())
